src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,62 @@
+.menu {
+  height: 100vh;
+  background-color: #ECECEC;
+  opacity: 0;
+  font-size: 22px;
+  letter-spacing: 2px;
+
+  overflow: auto; /* to scroll the menu if needed */
+
+  // &__top {
+  //   position: sticky; /* to keep cross always visible */
+  //   top: 0;
+  //   z-index: 1;
+  //   background-color: #e5e5e5;
+  // }
+
+  transition: opacity ease 0.3s, transform ease 0.6s;
```

Comment:
If you're using SCSS, consider using variables for the transition durations for easier maintenance.